src/server/system_services/schemas/account_schema.js

```diff
@@ -27,7 +27,14 @@ module.exports = {
         has_login: { type: 'boolean' },
         password: { wrapper: SensitiveString }, // bcrypted password - DEPRECATED
         next_password_change: { date: true }, // DEPRECATED
-
+        // TODO: Check NSFS behaviour for account and iam user
+        owner: { objectid: true },
+        tagging: {
+            $ref: 'common_api#/definitions/tagging',
+        },
+        is_iam: { type: 'boolean' },
+        iam_arn: { type: 'string' },
+        iam_path: { type: 'string' },
```

Comment:
I would add a comment (copied from NC)
```suggestion
        // iam_path - AWS IAM path (identifier)
        iam_path: { type: 'string' },
```